Updates content/es/*.{json,md}

stringparser opened the pull request:
Updates from @iojs/iojs-es 

From the commits below, what is left is to update is the link to the [latest weekly-update](https://medium.com/node-js-javascript/io-js-week-of-march-6th-2f9344688277) which is [published already in medium](https://medium.com/@iojs_es/io-js-semana-del-6-de-marzo-82739753062e). I'll update it on a second. Also, if you want it I'll squash them down.

snostorm commented:
Thanks @stringparser! We'll need a +1 from @iojs/website 

_Also, for whoever ends up merging this, let's use a squash._

stringparser commented:
No problem @snostorm :). For the moment the the index.md links to [medium.com/@iojs-es](https://medium.com/@iojs_es/io-js-semana-del-6-de-marzo-82739753062e)

snostorm commented:
Cool. Just waiting on the +1.

stringparser commented:
ping! :)

Fishrock123 commented:
@stringparser I think we are waiting another person from @iojs/iojs-es to give a +1

stringparser commented:
@Fishrock123 yes, I forgot to mention. I've also talked to them by chat. Thanks.

stringparser committed:
align content/es/*.{json,md} with content/en/*.{json,md}:
 - index.md: change link for the weekly update
 - faq.md: heading for a `why` section
 - es6.md: ES6 version, line break for `WeakSet` link, --es_staging flag descriptions
 - template.json: added fields, translated api-link
 - review: added @ipeluffo suggestions for index.md (iojs-es/website#3) and es6.md (iojs/website#285)

stringparser commented:
@ipeluffo, changes you suggested done, let me know if anything else is needed

ipeluffo commented on content/es/es6.md:
```diff
@@ -51,10 +53,21 @@ Puedes ver una lista más detallada, incluyendo la comparación con otros motore
 
-* [Clases](https://github.com/lukehoban/es6features#classes) (sólo en strict mode)
-* [Object literal extensions](https://github.com/lukehoban/es6features#enhanced-object-literals)
+* [Clases](https://github.com/lukehoban/es6features#classes) (sólo en strict mode, utilizando la flag `--harmony_classes` lo que implica que el block scope y object literal extensions)
```
I would replace:
"... lo que implica que el block scope ..." by "... lo que implica block scope ..."

Also, Do you think the expression for flag should be "la flag" o "el flag"?

Lastly, I think that if we replace "tras el/la flag" by "utilizando el/la flag", we should replace that expression in all the document :)

stringparser replied:
Ok, perfect. I would also drop the article for "flag" as you did for "el block" -> "block". And yes, I'll replace it anywhere. I always grep the document for laziness anyway :D, but thanks.

stringparser replied:
Arg, when block scope is with a verb before there is no problem to drop the article but for flag we have to use an article. I'll use "la" since is the Spanish gender of "bandera".

stringparser committed:
corrections:
- lo que implica block scope/implica block scope
- la flag, pick feminine as gender since `bandera` is feminine in spanish

julianduque commented:
it's a +1 from me

ipeluffo commented:
+1 from me too

snostorm commented:
Landing. Thanks all.

snostorm commented:
Published via https://github.com/iojs/website/commit/b7aed09ede48021477e68afbe97ee7ef084b48c2
